packages/long-short-pair/scripts/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+NETWORK=$1
+SUBGRAPH_NAME=$NETWORK'-lsp'
```

Comment:
Should I create these subgraphs under `nicholaspai`? @mrice32 has to make them under `umaprotocol`

Reply:
it would be great if the staging could be under `umaprotocol` as well. how can @nicholaspai and I get access to the subgraph `umaprotocol`? Once created in their dashboard we really just need the API keys, yes?

Reply:
yeah we have the api keys but we can't create new subgraphs unless we're admins. This is the main reason why I use the `nicholaspai` namespace